src/answers-umd.js

```diff
@@ -388,6 +389,19 @@ class Answers {
     this.core.globalStorage.set('queryTrigger', 'initialize');
     this.core.setQuery(searchConfig.defaultInitialSearch);
   }
+
+  /*
+   * Updates the css styles with new current variables. This is useful when the css
+   * variables are updated dynamically (e.g. through js).
+   * @param {Object} config Additional config to pass to the polyfill
+   */
+  polyfillCssVariables (config) {
```

Comment:
Do we need to give users the ability to pass in config here? I think it would be better for us to prescribe how cssVars should be invoked when used with the SDK. A client would need to read the css-vars-ponyfill docs to know what to do here. If they made a mistake, it could cause some issues with their site. It doesn't seem like they'll really need to configure the call to cssVars, provided we set it up correctly.

Reply:
Updated to only allow callbacks!